docs/src/manifests/manifest.json

```diff
@@ -13,6 +13,11 @@
           "title": "Quick Start",
           "path": "/quick-start",
           "editUrl": "/quick-start.md"
+        },
+        {
+          "title": "Articles",
```

Comment:
"Articles" terlalu generic gk? Mungkin bisa lebih deskriptif? How-Tos? Self Help Corner? Knowledge base?

Reply:
Need to confirm with @nicnocquee first :wink:

Reply:
Self help corner sounds like monika is too depressing to use that we need to have a section called that. 😅 Knowledge base usually for FAQs I think. I think Articles is ok for now.